apps/web/src/lang/fr.json

```diff
@@ -3190,14 +3194,14 @@
     "defaultMessage": "Veuillez nous faire part de prix et de reconnaissances en particulier que vous avez reçus en raison du fait que vous avez été au-delà de votre devoir.",
     "description": "Description for award experience section"
   },
+  "IWWSkL": {
+    "defaultMessage": "Traiter l’information",
```

Comment:
I believe this is incorrect. The original string, Process information, is using the word process as a noun (**a process**) where as _traiter_ is the verb (**to process**). If my assumption is correct, the string for `R5sGKY` would also need to be changed.